terraform/files/bin/deploy_harbor.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+. ~/bin/cccfg.inc
+export KUBECONFIG=${KUBECONFIG_WORKLOADCLUSTER}
+export OS_CLOUD=$PREFIX-$CLUSTER_NAME
+
+# export harbor variables as env for envsubst
+set -a
+. ~/$CLUSTER_NAME/harbor-settings
+set +a
+
+echo "Deploy harbor to $CLUSTER_NAME"
+
+mkdir -p ~/$CLUSTER_NAME/deployed-manifests.d/harbor
+cd ~/$CLUSTER_NAME/deployed-manifests.d/harbor
+
+# download scripts
+if test ! -s ~/kubernetes-manifests.d/harbor/harbor-secrets.bash; then
+  curl -L https://raw.githubusercontent.com/SovereignCloudStack/k8s-harbor/main/base/harbor-secrets.bash -o ~/kubernetes-manifests.d/harbor/harbor-secrets.bash || exit 2
+fi
+if test ! -s ~/kubernetes-manifests.d/harbor/s3-credentials.bash; then
+  curl -L https://raw.githubusercontent.com/SovereignCloudStack/k8s-harbor/main/envs/public/s3-credentials.bash -o ~/kubernetes-manifests.d/harbor/s3-credentials.bash || exit 2
```

Comment:
Is there a better way to get this, maybe as a submodule or porting it to our repository? I don't like much downloading it from main branch.

Reply:
What do you think about fixing it on a specific commit?

Reply:
I like that more. However, I would propose writing in some comments why you chose that commit, e.g. release X.Y. and maybe a link to that commit not just raw?

Reply:
I tagged k8s-harbor repo and used that tag instead of commit, so it will be more readable.
See 4cc002a47e150192f43fc534dbd82e5dd9c3ae2c